Gnt ci

canonici opened the pull request.

redianthus committed:
update CI

Commit:
dummy gwdb

michelmno commented:
Le rajout de https://github.com/geneweb/geneweb/pull/1853/commits/1c0a322270f9029a3f558e0be1ff8d4d1299a975  devrait être fait après la correction du  problème #1793 ! c'est ce que j'avais suggéré avec le PR  #1839

ilankri reviewed (commented):
Could you please split the first three commits (and the others if needed)?

Commit:
don't run mergeInd test

Commit:
add alcotest dependency

Commit:
test_place => place_test with alcotest

Commit:
test_calendar => test/date_test using alocatest

Commit:
test_sosa => sosa_test with alcotest

Commit:
test_wiki => wiki_test with alcotest

Commit:
test_utils => util_test with alcotest

Commit:
remove test_chronology

Commit:
test_ports => ports_test with alcotest

Commit:
allow to build using dummy_gwdb

Commit:
rm test_mergeInd

Commit:
skip known failures

Commit:
fix asset

ilankri committed:
Cleanup: Simply an expression

ilankri committed:
Revert "rm test_mergeInd"

This reverts commit f5c718f90d84c795f1d335181091245e97474a1b.

ilankri committed:
Cleanup: Delete a trailing blank line

ilankri committed:
mergeInd.mli: Undo faulty code removal

Undo bcce0b3573fc8417d0d30c5b1154ca03a465d90f on `lib/mergeInd.mli`.

The function `is_ancestor` needs to be exported for the tests.

ilankri committed:
Cleanup: Get rid of some top-level `open`s

ilankri committed:
test: Port `test_mergeInd` to Alcotest

ilankri committed:
Cleanup: Dispose of a call to `Obj.magic`

ilankri committed:
test: Make `Gwdb_driver.make` handle empty base name like legacy one

ilankri committed:
test: Cache the ascends in `test_mergeInd`

ilankri committed:
Cleanup: Use `()` instead of `_` for a `unit` parameter

ilankri committed:
test: Fix some typos

ilankri committed:
Cleanup: Use consistent naming for all tests
